utils/casadm.8

```diff
@@ -460,7 +460,7 @@ Identifier of cache instance <1-16384>.
 
 .TP
 .B -d, --core-device <DEVICE>
-Path to core device e.g. HDD device.
+Path to core device e.g. HDD device (/dev/disk/by-id/wwn-0x1234567890b100d).
```

Comment:
Maybe we should state "by-id" requirement here explicitly: 
Path to core device using by-id link (e.g. /dev/disk/by-id/wwn-0x1234567890b100d).

Reply:
thx